web/src/app/modules/shared/pipes/filterDeleted/filter-deleted.pipe.ts

```diff
@@ -0,0 +1,9 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { TableRowWithMetadata } from '../../components/presentation/datagrid/datagrid.component';
+
+@Pipe({ name: 'filterDeleted', pure: true })
+export class FilterDeletedPipe implements PipeTransform {
```

Comment:
Would it be better if this had a better name? I will suggest `FilterDeletedDatagridRowPipe`